src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -457,7 +457,7 @@ std::string statisticsTreat
 
   if (noOfDroppedNotifications != -1)
   {
-    out += TAG_ADD_COUNTER("droppedNotifications", noOfDroppedNotifications);
+    out += TAG_ADD_COUNTER("droppedNotifications", noOfDroppedNotifications + 1);
```

Comment:
I'd suggest to add a comment about why the +1 is neded. Just something like, "Given that the noOfDroppedNotifications  starts at -1, a +1 adjustement is needed".